extensions/amp-consent/0.1/test/test-consent-state-manager.js

```diff
@@ -376,6 +384,22 @@ describes.realWin('ConsentStateManager', {amp: 1}, (env) => {
           expect(storageSetSpy).to.not.be.called;
           expect(storageRemoveSpy).to.be.calledOnce;
         });
+
+        it('allows large consentInfo when not using viewer storage API', async () => {
```

Comment:
since the descriptions says viewer storage API is not used specifically here. 
Could you make `usesViewer=false` by default, and sets its value to true here instead